RadixWallet/Features/AccountPreferencesFeature/Children/ThirdPartyDeposits/ThirdPartyDeposits+Reducer.swift

```diff
@@ -155,10 +156,6 @@ struct ManageThirdPartyDeposits: FeatureReducer, Sendable {
 				}
 
 			} catch {
-				/// Polling failure will be displayed in SubmiTransactionView
```

Comment:
Should this go away?

Reply:
yes! we no longer need to catch the error, since the client will now return `TransactionStatus.permanentlyRejected` (rather than throw `TXFailureStatus.permanentlyRejected` as it used to do). 

Then, inside [SubmitTransaction](https://github.com/radixdlt/babylon-wallet-ios/pull/1368/files/12cb70b31f4db067b9a851f444440a080fb950eb#diff-892df0bbb6ddf6c18999e02fb9cd9942360dcbe2c84cb787b672bbf870d87c01R133), we react the same as we did, but we no longer have to add this "if error thrown correspond to a transaciton failure, then ignore error" kind of logic in every reducer that submits a transaction :)